Python/Product/PythonTools/PythonTools/Repl/InteractiveWindowProvider.cs

```diff
@@ -91,13 +94,27 @@ class InteractiveWindowProvider {
             return false;
         }
 
-        public IVsInteractiveWindow Open(string replId) {
+        public void OnWindowUsed(IVsInteractiveWindow window) {
```

Comment:
Does it have to be public?

Reply:
It's a private class, but the point is that it may be used from elsewhere when we "use" an interactive window.

Reply:
Hypothetically, we could reorder windows every time you use one, or switch focus to it. Neither of these requires "getting" the window from the provider, but it does require the provider to know that it's been used.